docs/users/fbc_onboarding.md

```diff
@@ -97,19 +94,38 @@ catalogs
 Artifacts generated in the previous step need to be added to a git and submitted via pull request. The operator pipeline validates the content of the catalogs and releases changes into ocp catalogs.
 
 ```bash
-$ git add operators/aqua/{catalog-templates,catalogs.yaml,composite-config.yaml,ci.yaml}
+$ git add operators/aqua/{catalog-templates,ci.yaml}
 
-$ git add
 $ git add catalogs/{v4.12,v4.13,v4.14,v4.15,v4.16}/aqua
 
 $ git commit --signoff -m "Add FBC resources for aqua operator"
 ```
 
 ## Generating catalogs from templates
-Catalog templates are used to simplify a view of a catalog and allow easier manipulation of catalogs. The automated conversion pre-generates a basic + composite template that can be turned into full FBC using the following command:
+Catalog templates are used to simplify a view of a catalog and allow easier manipulation of catalogs. The automated conversion pre-generates a basic template that can be turned into full FBC using the following command:
 
 ```bash
-opm alpha render-template composite -f catalogs.yaml -c composite-config.yaml
+opm alpha render-template basic ./catalog-templates/v${VERSION}.yaml > ${ROOT}/catalogs/v${VERSION}/${OPERATOR_NAME}/catalog.yaml
 ```
+For the example `aqua` operator above, if could look like:
+```bash
+ROOT=../..
+for VERSION in "4.12 4.13 4.14 4.15 4.16"
+do
+   opm alpha render-template basic ./catalog-templates/v${VERSION}.yaml > ${ROOT}/catalogs/v${VERSION}/aqua/catalog.yaml
+done
+```
+
+# -------------------------------------------------------------------
```

Comment:
Sorry for the specific call-out here.  I don't know if you're the right person @Allda.  I got distracted from working on this a couple of times today, so I put the note in here so I wouldn't lose it. 😁

Reply:
We already have a tracker for it and we will consolidate the documentation and the tooling (makefile) to provide a single way of interacting with catalogs and onboardings.

Could you please remove the comment so we can potentially merge it without the notes?